lib/osf-components/addon/components/file-browser/delete-file-modal/template.hbs

```diff
@@ -0,0 +1,103 @@
+            {{t 'osf-components.file-browser.delete_file_modal.delete_success' itemCount=this.deletedFiles.length}}
+        {{else}}
+            {{t 'osf-components.file-browser.delete_file_modal.heading' itemCount=@items.length}}
+        {{/if}}
+    </dialog.heading>
+    <dialog.main>
+        <ul local-class='ItemList'>
+            {{#if this.shouldShowFailureModal}}
+                {{#each this.failedFiles as |item|}}
+                    <li local-class='ListItem'>
+                        <div>
+                            {{#if item.isFile}}
+                                <FaIcon @icon='file-alt' />
+                            {{else}}
+                                <FaIcon @icon='folder' />
+                            {{/if}}
+                            <span local-class='ItemName'>{{item.name}}</span>
+                        </div>
+                        <Button
+                            @layout='fake-link'
+                            {{on 'click' (perform this.retryDelete item)}}
+                        >
+                            <FaIcon @icon='undo' />
+                        </Button>
+                    </li>
+                {{/each}}    
+            {{else if this.shouldShowSuccessModal}}
+                {{#each this.deletedFiles as |item|}}
+                    <li local-class='ListItem'>
+                        <div>
+                            {{#if item.isFile}}
+                                <FaIcon @icon='file-alt' />
+                            {{else}}
+                                <FaIcon @icon='folder' />
+                            {{/if}}
+                            <span local-class='ItemName'>{{item.name}}</span>
+                        </div>
+                        <FaIcon local-class='CheckIcon' @icon='check' />
+                    </li>
+                {{/each}}
+            {{else}}
+                {{#each @items as |item|}}
+                    <li local-class='ListItem'>
+                        <div>
+                            {{#if item.isFile}}
+                                <FaIcon @icon='file-alt' />
+                            {{else}}
+                                <FaIcon @icon='folder' />
+                            {{/if}}
+                            <span local-class='ItemName'>{{item.name}}</span>
+                        </div>
+                    </li>
+                {{/each}}
+            {{/if}}
+        </ul>
+    </dialog.main>
+    <dialog.footer>
+        {{#if (or this.shouldShowFailureModal this.shouldShowSuccessModal)}}
+            <Button
+                @type='secondary'
+                {{on 'click' (
+                    queue
+                    this.reset
+                    dialog.close
+                )}}
+            >
+                {{t 'general.done'}}
+            </Button>
+        {{else}}
+            <Button
+            @type='secondary'
+            {{on 'click' dialog.close}}
```

Comment:
When a user hits cancel, should we stop the delete tasks that are currently in progress? Also needs one more indent level